deploy_and_populate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I'd like 2 comments put in this file here:
 - why is this different from other `mix` incantations which went inline into `launcher.py` - unless the reason I'm missing is obvious
 - a loud remark that `DevHelpers` - and this by consequence - is not intended for production (a `# TODO` ideally)